### DISCUSSION TITLE
How can I see what percentage of the metabolic process a particular enzyme occupies?

### DISCUSSION THREAD

**Discussion by RoachWang** [Open-Systems-Pharmacology/Forum#1615] [Questions & Problems] [answered]
I would like to know which one I should choose if I wish to see the percentage of a particular enzyme, occupied in the metabolism of a drug? Organism or liver？
If the Liver tab is selected, which sub-tab should I continue to select?Blood cell？Interstitial？Or intarcellular？
What is the difference between the meanings represented by the Organism and the sub-options in liver options?
And，In pksim, is the sum of the proportions of the elimination processes in a model equal 100% by default?
![微信截图_20230911174525](https://github.com/Open-Systems-Pharmacology/Forum/assets/139726222/0582a765-1feb-4787-91eb-bb212bc241f0)

**Comment by tobiasK2001** [comment, marked as answer]
Hi,

if you want to see fraction metabolized by CYP3A4 in the whole organism. (e.g. in liver plus intestine) select Organism. If you want to see what is metabolized in Liver only : select liver intracellular. That is where the enzyme CYP3A4 is located. Interstitial would return zero of course as no CYP3A4 is there. PK-Sim assumes that the user is knowing where the metabolism takes place and does chose the right sub-compartment ;-)

Hope that might help a bit.

Best, Tobias

**Reply by RoachWang** [reply]
Thank you for your answers, but I still have some basic questions I'd like to ask (please forgive my weak basic knowledge)
When described in the literature as "After dosing in humans, the following compounds (expressed as % of the dose) were excreted in the urine: N-desethyl levomilnacipran (18.2%)". I want to simulate this "18.2%" in PKsim, should select the liver tab, right? After all, my personal understanding is that only drugs that enter the systemic circulation are excreted in the urine.
As well as descriptions such as " . . is a major metabolic pathway accounting for approximately 18 % of total clearance", and the expression above - ".... . excreted in the urine..." -- can be considered to have the same meaning?

**Comment by xiangetang** [comment]
I occoured the similar problem, my PK-sim cannot show the fraction of dose for respective enzyme. How do I do? Thanks very much!
![bd9e2bf8fee9514deb85e171f2d48aa](https://github.com/Open-Systems-Pharmacology/Forum/assets/98267394/08ebf6ac-2a45-46f7-ac03-8f26d4103ea8)

**Reply by ljiaqian2020-cloud** [reply]
Did you ever find a solution for this?
